W-21230689-doc-feedback-disconnecting-portal-ks (#87)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/disconnecting-the-portal.adoc

include::_attributes.adoc[]
endif::[]

> Anypoint can connect to only one Salesforce organization, and you can create only one API portal for that organization. To create a new API portal, you must disconnect the current API portal from your Salesforce organization. This process unlinks Anypoint from your Salesforce organization. You can then link your Anypoint account to a new Salesforce organization.

Reviewer: ```suggestion
Anypoint Platform can connect to only one Salesforce organization, and you can create only one API portal for that organization. To create a new API portal, you must disconnect the current API portal from your Salesforce organization. This process unlinks Anypoint Platform from your Salesforce organization. You can then link your Anypoint Platform account to a new Salesforce organization.
```

Reviewer: Global: don't use Anypoint alone.

[[what-to-consider]]
== Before You Disconnect

> Disconnecting your API portal removes artifacts from the API Experience Hub database and unlinks Anypoint from your Salesforce organization. The following artifacts are permanently removed from API Experience Hub:

Reviewer: ```suggestion
Disconnecting your API portal removes artifacts from the API Experience Hub database and unlinks Anypoint from your Salesforce organization.

These artifacts are permanently removed from API Experience Hub:
```

* All published assets and their visibility settings
* All custom user groups
* All group mapping configurations (for both custom and default user groups)
* The Single Sign-On (SSO) configuration

[[if-you-reconnect]]

> === If You Reconnect Later

Reviewer: I wonder if this could be combined in the intro under the H1.

> If you reconnect the portal (for example, after creating a new connection), you must reconfigure SSO, re-publish all APIs to the portal, and reconfigure group mapping. There is no process to restore the previous configuration automatically. Note that styling metadata is retained when reconnecting to the same Salesforce organization.

Reviewer: ```suggestion
If you reconnect the portal (for example, after creating a new connection), you must reconfigure SSO, republish all APIs to the portal, and reconfigure group mapping. There is no process to restore the previous configuration automatically. Note that styling metadata is retained when reconnecting to the same Salesforce organization.
```

Reviewer: It might be worth mentioning this in the disconnect task - that you can't restore automatically.

Reviewer: nm, I guess this is all part of "Before You Disconnect".

[[changing-connection-user]]

> === Changing the User Who Created the Connection

Reviewer: Maybe this content goes in the intro?

> There is no supported way to change the user who created the connection in API Experience Hub. To assign the connection to a different user, you must disconnect and create a new connection. For more information, see xref:assigning-permissions.adoc[Assigning Permissions].

Reviewer: If you don't hard-code the link text, it will be picked up automatically.

[[before-disconnect-steps]]

> === Steps to Complete Before Disconnecting

Reviewer: ```suggestion
=== Before You Disconnect the Portal
```

Before you disconnect, record the following information so you can restore your portal configuration after reconnecting:

. Record your published APIs and visibility settings.
.. From the *API management* page, select the *APIs in the portal* tab.

> .. Record a list of all APIs in the portal. You must re-publish these under the new connection.

Reviewer: ```suggestion
.. Record a list of all APIs in the portal.
+
You must republish these APIs under the new connection.
```

> .. For each API, record its visibility settings (including instance and user visibility per version) so you can restore them after reconnecting. For more information, see xref:managing-apis.adoc#make-api-versions-visible[Make API Versions Visible].

Reviewer: ```suggestion
.. For each API, record its visibility settings (including instance and user visibility per version) so you can restore them after reconnecting.
+
For more information, see xref:managing-apis.adoc#make-api-versions-visible[Make API Versions Visible].
```

Reviewer: (I use a line break for easier scanning of the actual step.)

. Record your user groups information.

> .. From the *User groups* page, record all user groups and their details (for example, group names and members). For more information, see xref:managing-users.adoc#manage-user-groups[Manage User Groups].

Reviewer: ```suggestion
.. From the *User groups* page, record all user groups and their details (for example, group names and members).
+
For more information, see xref:managing-users.adoc#manage-user-groups[Manage User Groups].
```

> .. For each group, record the corresponding identity provider name. You will need this when you re-enable SSO and reconfigure group mapping. For more information, see xref:managing-users.adoc#map-user-groups-to-external-identity-providers[Map User Groups to External Identity Providers] and xref:configuring-sso.adoc[Configuring SSO].

Reviewer: ```suggestion
.. For each group, record the corresponding identity provider name.
+
You need this information when you re-enable SSO and reconfigure group mapping. For more information, see xref:managing-users.adoc#map-user-groups-to-external-identity-providers[Map User Groups to External Identity Providers] and xref:configuring-sso.adoc[].
```

== Disconnect the Portal

. From the *Manage your API portal* page, click *Settings*.
. Click *Disconnect Salesforce Org*.
. To confirm, click *Disconnect Salesforce Org* in the message.

== See Also

> * xref:connecting-to-salesforce.adoc[Connecting to Salesforce]

Reviewer: Unless you're changing the link text, leave it blank for autofill. It's easier to maintain - if the target headings change, they populate automatically.
